include/sta/ConcreteLibrary.hh

```diff
@@ -38,6 +38,7 @@ class LibertyCell;
 class LibertyPort;
 
 typedef Map<const char*, ConcreteCell*, CharPtrLess> ConcreteCellMap;
+typedef Map<std::string, const char*> AttributeMap;
```

Comment:
why not string -> string?
20 years ago std::string was pretty worthless, but I am trying to make more use of it these days

Reply:
Just to double check, you're okay with moving all the method signatures to std::string including get and set attribute?

Reply:
I was suggesting changing the map so managing the deletes would be easier.
I think you can get away with changing the get/set methods and rely on automatic coercion when const char*'s are passed in. Sometime I just try it can see what feels cleaner.
So ya, I am ok with it if it doesn't end getting messy.